src/modules/70-pipeline/components/ArtifactsSelection/ArtifactUtils.ts

```diff
@@ -277,6 +278,19 @@ const getTagValues = (
   if (specValues?.tag && getMultiTypeFromValue(specValues?.tag) === MultiTypeInputType.FIXED) {
     values.tag = { label: specValues?.tag, value: specValues?.tag }
   }
+
+  return values
+}
+
+export const getDigestValues = (specValues: any): ImagePathTypes => {
```

Comment:
`getDigestValues` is used here only in this files. So we can make this normal funtions.
```suggestion
const getDigestValues = (specValues: any): ImagePathTypes => {
```